src/routes/docs/quick-starts/dart/+page.markdoc

```diff
@@ -87,56 +87,56 @@ Future<void> main() async {
 {% /section %}
 {% section #step-5 step=5 title="Initialize database" %}
 
-Once the Appwrite Client is initialized, create a function to configure a todo collection.
+Once the Appwrite Client is initialized, create a function to configure a todo table.
 
 ```dart
 var databases;
 var todoDatabase;
-var todoCollection;
+var todoTable;
 
 Future<void> prepareDatabase() async {
-  databases = Databases(client);
+  tablesDb = TablesDb(client);
 
-  todoDatabase = await databases.create(
+  todoDatabase = await tablesDb.createDatabase(
```

Comment:
lets make sure `createDatabase` doesn't exist anywhere for `TablesDb` API, cc @adityaoberai